libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -340,9 +340,9 @@ void AP_DDS_Client::update_topic(sensor_msgs_msg_BatteryState& msg, const uint8_
     if (battery.current_amps(current, instance)) {
         if (percentage == 100) {
             msg.power_supply_status = 4;   //POWER_SUPPLY_STATUS_FULL
-        } else if (current < 0.0) {
+        } else if (is_negative(current)) {  // TODO
```

Comment:
```suggestion
        } else if (is_negative(current)) {
```

What is the TODO, this looks like a good change to me